browser/templates/browser/investigator.html

```diff
@@ -0,0 +1,109 @@
+
+    </div>
+    {% if person.authority %}
+    <div class="text-warning">
+        <span class="glyphicon glyphicon-barcode"></span>
+        {{ person.authority.conceptpower_uri|urlize }}
+    </div>
+    {% endif %}
+    {% if person.validated %}
+    <div>
+        <span class="glyphicon glyphicon-ok" style="color:green;"></span>&nbsp;
+        Validated by {{ person.validated_by }} on {{ person.validated_on }}.
+    </div>
+    {% endif %}
+</div>
+
+<div class="container">
+    <p class="text-info">
+        {% if type == 'edit_investigator' %}
+        The below form will edit this <strong>Investigator</strong> record for <strong>{{ person.name }}</strong>.
+        {% else %}
+        This form will create a new <strong>Investigator</strong> record with the information provided below. The
+        information will be added to <strong>{{ person.name }}</strong>'s record.
+        {% endif %}
+    </p>
+
+    <form action="." method="post" id="InvestigatorForm">
+    {% csrf_token %}
+    {% block form_fields %}
+    {% with form.subject as field %}
+    <div class="row">
+        <div class="col-xs-12">
+        <div class="form-group">
+            {% for error in field.errors %}
+            <div class="alert alert-warning">{{ error }}</div>
+            {% endfor %}
+            <label>{{ field.label }}</label>
+            {{ field|addcss:"form-control input-sm" }}
+            <p class="text-muted">{{ field.help_text }}</p>
+        </div>
+        </div>
+    </div>
+    {% endwith %}
+
+    {% with form.role as field %}
+    <div class="row">
+        <div class="col-xs-8">
+        <div class="form-group">
+            {% for error in field.errors %}
+            <div class="alert alert-warning">{{ error }}</div>
+            {% endfor %}
+            <label>{{ field.label }}</label>
+            {{ field|addcss:"form-control input-sm" }}
+            <p class="text-muted">{{ field.help_text }}</p>
+        </div>
+        </div>
+    {% endwith %}
+
+    {% with form.year as field %}
+        <div class="col-xs-4">
+        <div class="form-group">
+            {% for error in field.errors %}
+            <div class="alert alert-warning">{{ error }}</div>
+            {% endfor %}
+            <label>{{ field.label }}</label>
+            {{ field|addcss:"form-control input-sm" }}
+            <p class="text-muted">{{ field.help_text }}</p>
+        </div>
+        </div>
+    </div>
+    {% endwith %}
+    {% endblock %}
+
+    <div class="row">
+    <div class="text-center">
+        {% if type == 'edit_investigator' %}
+        <input class="btn btn-success" type="submit" form="InvestigatorForm" name="UpdateButton" id="ValueButton" value="Update">
+        {% else %}
+        <input class="btn btn-success" name="SaveButton" form="InvestigatorForm" type="submit" value="Save">
+        {% endif %}
+        <input class="btn btn-danger" name="CancelButton" type="button" id="CancelButton" value="Cancel">
```

Comment:
this can simply be a link (with the correct css class to make it look like a button) and then use django template url to create the link (no funky javascript substring hack).

Reply:
Changed the CancelButton to a link. The css gives it the appearance of a button and the link is handled through Django template